apps/settings/style/settings.css

```diff
@@ -918,13 +923,12 @@ ul li > label.range-icons.volume input {
   height: auto;
 }
 
-#feedback-happy {
-  float: left;
-}
+html[dir="ltr"] #feedback-happy { float: left; }
```

Comment:
might worth to add a comment /* can replace with float: inline-end once its support */